### PR TITLE
Please review setup file changes

### PR DESCRIPTION
**Pull request by shubham-joshi**

### CODE REVIEW

**Review by fs-projects — APPROVED** [review]

**Review by sjoshi93 — COMMENTED** [review]
Please let me know if username is correct.

**Review by sjoshi93 — CHANGES_REQUESTED** [review]
Please let me know if username is correct?

**Review by utkarsh-kukreti-del — APPROVED** [review]

**Comment by shubham-joshi** [comment]
> Please let me know if username is correct?

Yes, username is correct.

File: setup.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 Initial setup commit
+Second line shubham-joshi
```
Comment (sjoshi93) [inline setup.txt:2]: This line doesn't seem to be right.

**Review by sjoshi93 — CHANGES_REQUESTED** [review]

File: setup.txt (hunk 1 above)
Comment (shubham-joshi) [reply setup.txt:2]: It is fine.

**Comment by shubham-joshi** [comment]
Generic comment has been fixed.

**Review by sjoshi93 — APPROVED** [review]
Looks good.

**Pull request merged**